.github/workflows/test_build.yml

```diff
@@ -32,6 +32,8 @@ jobs:
     if: github.repository == 'Macaulay2/M2' || contains(github.ref, 'global')
     name: ${{ matrix.build-system }}-${{ matrix.os }}-${{ matrix.compiler }}
     runs-on: ${{ matrix.os }}
+    env:
+      MakeDocumentation: false
```

Comment:
Why? Part of the tests is checking that packages install properly, including producing the documentation. In particular, one of the CMake builds actually creates a debian package which I use for testing.

Reply:
This is just temporary to make sure that this branch does the intended thing.  I'll revert it once everything looks good.